src/bci_build/package/__init__.py

```diff
@@ -1379,11 +1387,20 @@ def image_type(self) -> ImageType:
     def build_tags(self) -> List[str]:
         tags = []
         for name in [self.name] + self.additional_names:
-            tags += [
-                f"{self._registry_prefix}/bci-{name}:%OS_VERSION_ID_SP%",
-                f"{self._registry_prefix}/bci-{name}:{self.version_label}",
-            ] + (
-                [f"{self._registry_prefix}/bci-{name}:latest"] if self.is_latest else []
+            tags += (
+                [
+                    f"{self._registry_prefix}/bci-{name}:%OS_VERSION_ID_SP%",
+                    f"{self._registry_prefix}/bci-{name}:{self.version_label}",
+                ]
+                + (
+                    [f"{self._registry_prefix}/bci-{name}:latest"]
+                    if self.is_latest
+                    else []
+                )
+                + (
+                    [f"{self._registry_prefix}/bci-{name}:{tag}"]
+                    for tag in self.other_tags or []
```

Comment:
Needs a reformat:
```suggestion
                    [f"{self._registry_prefix}/bci-{name}:{tag}"
                    for tag in self.other_tags or []]
```

Reply:
Not sure if you still want the `other_tags`.